source/nodejs/tests/ui-tests/src/testUtils.ts

```diff
@@ -13,6 +14,7 @@ export class TestUtils {
     async goToTestCase(testCaseName: string): Promise<void> {
         const elementLinkText: Webdriver.WebElement = await this.driver.findElement(Webdriver.By.id(testCaseName));
         await elementLinkText.click();
+        await this.delay(this.delayForAfterCardRendering);
```

Comment:
it's easy to introduce stability issue with `delay`. I prefer to use driver.wait if possible.
https://www.selenium.dev/documentation/webdriver/waits/

 For example:
```
await driver.wait(until.titleIs('webdriver - Google Search'), 1000);
```